CHANGELOG.md

```diff
@@ -1,6 +1,50 @@
+- Fixed an account change event not emitting correctly. ([#7463](https://github.com/brave/brave-ios/issues/7463))
+- Fixed transaction links to open correct block explorer irrespective of the active network. ([#7455](https://github.com/brave/brave-ios/issues/7455))
+- Fixed loading indicator not being removed during asset auto discovery. ([#7433](https://github.com/brave/brave-ios/issues/7433))
+
+### General
+
+- Added top page color sampling for status bar overlay view color. ([#7479](https://github.com/brave/brave-ios/issues/7479))
+- Added biometrics and passcode authentication for Sync settings. ([#7436](https://github.com/brave/brave-ios/issues/7436))
+- Updated copy for Safe Browsing setting. ([#7430](https://github.com/brave/brave-ios/issues/7430))
+- Updated Playlist URL bar icons and menus. ([#7422](https://github.com/brave/brave-ios/issues/7422))
+- Updated copy text in Playlist. ([#7353](https://github.com/brave/brave-ios/issues/7353))
+- Updated Playlist to use Async-Await. ([#7349](https://github.com/brave/brave-ios/issues/7349))
+- Updated SSL Certificate Validation to match all other browsers. ([#5543](https://github.com/brave/brave-ios/issues/5543))
+- Enabled monogram favicons to keep background color between app launches. ([#7396](https://github.com/brave/brave-ios/issues/7396))
+- Fixed VPN profile from failing to initialize after purchase. ([#7413](https://github.com/brave/brave-ios/issues/7413))
+- Fixed issues with cosmetic filters during slow page loads. ([#7612](https://github.com/brave/brave-ios/issues/7612))
+- Fixed Find In Page Bar from dismissing when touched in top-bar mode. ([#7610](https://github.com/brave/brave-ios/issues/7610))
+- Fixed a crash when closing tabs. ([#7548](https://github.com/brave/brave-ios/issues/7548))
+- Fixed various issues with CarPlay. ([#7541](https://github.com/brave/brave-ios/issues/7541))
+- Fixed https://youtube.com from adding audio files to playlist videos. ([#7537](https://github.com/brave/brave-ios/issues/7537))
+- Fixed tabs not being restored after closing private tab or toggling tab view. ([#7517](https://github.com/brave/brave-ios/issues/7517))
+- Fixed RewardsScript not running in Desktop mode on iPhone. ([#7512](https://github.com/brave/brave-ios/issues/7512))
+- Fixed preroll ads being shown as black screen when disabling “Ads Blocking” in Shields. ([#7501](https://github.com/brave/brave-ios/issues/7501))
+- Fixed Sync Chain deletion authentication problems. ([#7493](https://github.com/brave/brave-ios/issues/7493))
+- Fixed issue with background color for “Logins & Passwords” in dark mode. ([#7481](https://github.com/brave/brave-ios/issues/7481))
+- Fixed issue with Tabs Bar being hidden after using QR code scanner. ([#7476](https://github.com/brave/brave-ios/issues/7476))
+- Fixed playlist lock-screen pausing issues. ([#7466](https://github.com/brave/brave-ios/issues/7466))
```

Comment:
Should `playlist` be capitalized here? It's capitalized in certain instances in these notes and not in others.

Reply:
I think you're correct, will fix.